modules/atomic-widgets/module.php

```diff
@@ -74,4 +79,26 @@ private function enqueue_scripts() {
 			true
 		);
 	}
+
+	/**
+	 * Add the CSS for the element.
+	 *
+	 * @param Post $post
+	 * @param Element_Base $element
+	 */
```

Comment:
this PHPDoc doesn't help much (both in terms of descriptiveness and typings)
either explain properly or remove it completely